spec/ttml1.xml

```diff
@@ -1097,6 +1097,35 @@ conformant <emph>Document Instance</emph>, a given <emph>Document Instance</emph
 constrained by the author or authoring tool to satisfy a more
 restrictive definition of validity.</p>
 </note>
+<note>
+<p>As illustrated in the following example, an <emph>Abstract Document Instance</emph> can be a <emph>Valid Abstract Document Instance</emph> even if it includes elements and attributes whose namespace names are listed in <specref ref="namespace-vocab-table"/> but are not defined in this version of the specification. Specifically, the element <code>foo</code> and the attribute <att>tts:foo</att> are pruned by above steps (1) and (3), respectively, because they are not members of the associated <emph>Abstract Document Type</emph>, even though their namespace names are listed in <specref ref="namespace-vocab-table"/>.
```

Comment:
How can an element or attribute have a namespace name that is both listed in the table and not defined in the specification? Is that supposed to say that its namespace is listed in the table but its name is not defined within that namespace by this specification?

Reply:
The term "namespace name" is not used correctly here. Elements have a namespace or are "in" a namespace but do not have a namespace name. Therefore the following should be reworded

> if it includes elements and attributes whose namespace names are listed in ...

to 

> if it includes elements and attributes **in a namespace** listed in ...

At the end of the note it should read accordingly:

>  even though they are in a namespace listed in Table 1 – Namespaces.

Reply:
> Is that supposed to say that its namespace is listed in the table but its name is not defined within that namespace by this specification?

Yes.